src/SafeAreaContext.tsx

```diff
@@ -1,5 +1,6 @@
 import * as React from 'react';
-import { Dimensions, StyleSheet, ViewProps } from 'react-native';
+import { Dimensions, StyleSheet } from 'react-native';
```

Comment:
Can you add a type qualifier to the existing import instead of adding a 2nd one. Like import { A, type B } from …

Reply:
Done